charts/fybrik/files/taxonomy/base_taxonomy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "title": "taxonomy.json",
+  "definitions": {
+    "Action": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/ActionName"
+        }
+      },
+      "additionalProperties": true
+    },
+    "ActionName": {
+      "type": "string"
+    },
+    "AppInfo": {
+      "type": "object",
+      "additionalProperties": true
+    },
+    "AssetID": {
+      "description": "Asset ID of the asset to be queried in the catalog",
+      "type": "string"
+    },
+    "AttributeType": {
+      "type": "string",
+      "enum": [
+        "numeric",
+        "string",
+        "bool"
+      ]
+    },
+    "Capability": {
+      "type": "string"
+    },
+    "Connection": {
+      "description": "Name of the connection to the data source",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/ConnectionType",
+          "description": "Name of the connection to the data source"
+        }
+      },
+      "additionalProperties": true
+    },
+    "ConnectionType": {
+      "description": "Name of the connection type to the data source",
+      "type": "string"
+    },
+    "DataFlow": {
+      "type": "string",
+      "enum": [
+        "read",
+        "write",
+        "delete",
+        "copy"
+      ]
+    },
+    "DataFormat": {
+      "description": "Format of the data",
+      "type": "string"
+    },
+    "InfrastructureElement": {
+      "type": "object",
+      "required": [
+        "attribute",
+        "value",
+        "object"
+      ],
+      "properties": {
+        "arguments": {
+          "description": "A list of arguments defining a specific metric, e.g. regions for a bandwidth",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "attribute": {
+          "description": "Attribute name defined in the taxonomy",
```

Comment:
This is the taxonomy, so what does this mean?

Reply:
Changed the description